addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -1030,6 +1030,158 @@ There is no trigger anymore but one can use transformations to accomplish the sa
+
+Here a few examples of the Item configuration from the 1.x binding:
+
+The first Item polled with the first poller used this configuration (with offset 0):
+
+```
+Switch FooSwitch  "Foo Switch"  {modbus="slave1:0"} 
+```
+
+Now we have to define a ``Thing``that can later be bound to that Item.
+
+The ``slave1`` poller uses ``12288`` as start address. So we define the first data Thing within the poller ``wago_slave1`` with this address and choose a name that ends with ``0``:
+
+```
+Thing data wago_s1_000 [ readStart="12288", readValueType="bit", writeStart="12288", writeValueType="bit", writeType="coil" ]
+```
+
+The second Item of the 1.x binding (offset ``1``) is defined as follows.
+
+```
+Switch BarSwitch  "Bar Switch" {modbus="slave1:1"}
+```
+
+This leads to the thing definition
+
+```
+Thing data wago_s1_001 [ readStart="12289", readValueType="bit", writeStart="12289", writeValueType="bit", writeType="coil" ]
+```
+
+Note the absolute address ``12289`` (12288+1) which has to be used here.
+
+Incorporating this definitions into the thing file leads to:
+
+`wago.things`:
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+    Bridge poller wago_slave1 [ start=12288, length=128, refresh=500, type="coil" ] {
+        Thing data wago_s1_000 [ readStart="12288", readValueType="bit", writeStart="12288", writeValueType="bit", writeType="coil" ]
+        Thing data wago_s1_001 [ readStart="12289", readValueType="bit", writeStart="12289", writeValueType="bit", writeType="coil" ]
+    }
+
+    Bridge poller wago_slave2 [ start=12338, length=100, refresh=4000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave3 [ start=12438, length=100, refresh=5000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave4 [ start=12538, length=100, refresh=10000, type="holding" ] {
+    }
+}
+```
+
+Save this in the ``things`` subdirectory of your openHAB 2 config. Watch the file ``events.log`` as it lists your new added data ``Things``. Given that there are no config errors, they quickly change from ``INITIALIZING`` to ``ONLINE``. 
+
+Finally the Item definition has to be changed to refer to the new created ``data`` ``Thing``. You can copy the names you need for this directly from the ``events.log`` file:
+
+```
+Switch FooSwitch  "Foo Switch" {modbus="slave1:0"} 
+Switch BarSwitch  "Bar Switch" {modbus="slave1:1"}
+```
+
+turn into
+
+```
+Switch FooSwitch  "Foo Switch" {channel="modbus:data:wago:wago_slave1:wago_s1_000:switch", autopudate="false"} 
+Switch BarSwitch  "Bar Switch" {channel="modbus:data:wago:wago_slave1:wago_s1_001:switch", autoupdate="false"}
+```
+
+If the names used for the data ``Thing``s end with the numbers of the relative addresses in the former Item definition, it is possible to update even a huge amount of Item definitions with a few "search and replace" commands.
```

Comment:
single back ticks
`data` instead of plain data to match documentation elsewhere?